viteed_app/lib/screens/signup.dart

```diff
@@ -1,7 +1,18 @@
 import 'package:flutter/material.dart';
+import 'package:multi_select_flutter/multi_select_flutter.dart';
 import 'package:viteed_app/auth/authenticaton.dart';
 import '../constants/validators.dart';
 
+class Domain {
+  final int id;
+  final String name;
+
+  Domain({
+    required this.id,
+    required this.name,
+  });
```

Comment:
This constructor could be a const constructor

Reply:
@SirusCodes I'll add a const keyword over here. Thanks.